From: Josh Boyer <jwboyer@fedoraproject.org>
Date: Fri, 14 Nov 2014 09:17:39 -0500
Subject: [PATCH] drm/i915: Don't WARN in edp_panel_vdd_off

After talking with ajax, this isn't really worth splashing a backtrace
about.  The handling here is reworked in 3.18, but the fixes are a bit
larger than we want to backport.  Just shut it up for now.

Signed-off-by: Josh Boyer <jwboyer@fedoraproject.org>
---
 drivers/gpu/drm/i915/intel_dp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/gpu/drm/i915/intel_dp.c b/drivers/gpu/drm/i915/intel_dp.c
index 1b7375efc670..a6fb06cc6cf0 100644
--- a/drivers/gpu/drm/i915/intel_dp.c
+++ b/drivers/gpu/drm/i915/intel_dp.c
@@ -1303,8 +1303,6 @@ static void edp_panel_vdd_off(struct intel_dp *intel_dp, bool sync)
 	if (!is_edp(intel_dp))
 		return;
 
-	WARN(!intel_dp->want_panel_vdd, "eDP VDD not forced on");
-
 	intel_dp->want_panel_vdd = false;
 
 	if (sync)
-- 
2.1.0

